revvy/robot/remote_controller.py

```diff
@@ -134,15 +209,13 @@ def processing_time(self):
 
     @property
     def background_control_state(self):
-        return self._background_control_state
+        return self._background_control_state.get_numeric()
 
     @property
-    def control_button_pressed(self):
-        if self._control_button_pressed:
-            val = self._control_button_pressed
-            self._control_button_pressed = 0
-            return val
-        return 0
+    def fetch_autonomous_requests(self):
+        result = copy.copy(self._control_button_pressed)
```

Comment:
If anything, this functions should be part of the state "enum", IMO. Though I might just want to bring this piece of code closer to Rust, which is more familiar to me (where this is analogous to `Option::take()`) so feel free to disregard me.